### ISSUE TITLE
Ubuntu 22.04 - xcb plugin error and currentRouting bad type

### ISSUE THREAD

**Issue by Perlover** [LorenEteval/Furious#4]
Hello!

First run after installing by `pip install Furious-GUI`:

```
$ Furious 
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: minimal, eglfs, offscreen, minimalegl, wayland, wayland-egl, vnc, xcb, vkkhrdisplay, linuxfb.

qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: minimal, eglfs, offscreen, minimalegl, wayland, wayland-egl, vnc, xcb, vkkhrdisplay, linuxfb.
```

After googling I found that I need to run:

```
$ sudo apt install libxcb-cursor0
```

Then I got error from Furious:

```
$ Furious 
[2023-08-17 00:23:51,539] [Furious.Widget.Application] [INFO] application version: 0.2.0
[2023-08-17 00:23:51,542] [Furious.Widget.Application] [INFO] python version: 3.10.12.final.0. Platform: Linux
[2023-08-17 00:23:51,542] [Furious.Widget.Application] [INFO] system version: 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0]
[2023-08-17 00:23:51,542] [Furious.Widget.Application] [INFO] sys.executable: /usr/bin/python3
[2023-08-17 00:23:51,542] [Furious.Widget.Application] [INFO] sys.argv: ['/home/user/.local/bin/Furious']
[2023-08-17 00:23:51,542] [Furious.Widget.Application] [INFO] appFilePath: /usr/bin/python3.10
[2023-08-17 00:23:51,542] [Furious.Widget.Application] [INFO] system language is RU
[2023-08-17 00:23:51,542] [Furious.Widget.Application] [INFO] custom font Cascadia Mono load success
[2023-08-17 00:23:51,548] [Furious.Utility.Proxy] [INFO] turn off proxy success
[2023-08-17 00:23:51,548] [Furious.Utility.Proxy] [ERROR] turn on proxy daemon failed
[2023-08-17 00:23:51,633] [Furious.Widget.AssetViewer] [INFO] asset dir is '/home/user/.local/lib/python3.10/site-packages/Furious/Data/xray'
Traceback (most recent call last):
  File "/home/user/.local/lib/python3.10/site-packages/Furious/Widget/Application.py", line 293, in run
    return self.initTray().exec()
  File "/home/user/.local/lib/python3.10/site-packages/Furious/Widget/Application.py", line 250, in initTray
    self.editRoutingWidget = EditRoutingWidget()
  File "/home/user/.local/lib/python3.10/site-packages/Furious/Widget/EditRouting.py", line 996, in __init__
    self.editRoutingTableWidget = EditRoutingTableWidget(parent=self)
  File "/home/user/.local/lib/python3.10/site-packages/Furious/Widget/EditRouting.py", line 745, in __init__
    self.activateItemByIndex(routingToIndex(), activate=True)
  File "/home/user/.local/lib/python3.10/site-packages/Furious/Action/Routing.py", line 126, in routingToIndex
    index = int(currentRouting)
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
```

**Comment by LorenEteval** [comment]
Hi. Thanks for the report. This is fixed in 0.2.1. Please use `pip install --upgrade Furious-GUI` to upgrade and re-launch.

**Comment by LorenEteval** [comment]
Just a quick reminder: libxcb-cursor is indeed an extra dependency that added between PySide 6.4 and PySide 6.5, so yes it's the right way to get Furious started on Linux with PySide 6.5. Please let me know if there are futher issues.

**Comment by Perlover** [comment]
> Hi. Thanks for the report. This is fixed in 0.2.1. Please use `pip install --upgrade Furious-GUI` to upgrade and re-launch.

It works! Already the Furious was started and I am using through SOCKS5 in browser! Thank you!
